### DISCUSSION TITLE
Finding YARP in your environment

### DISCUSSION THREAD

**Discussion by Caradac** [dotnet/yarp#1652] [Q&A] [answered]
Given the new [security advisory](https://github.com/microsoft/reverse-proxy/security/advisories/GHSA-8xc6-g8xw-h2c4) for YARP, how should we go about identifying vulnerable versions in our environment?  

(Apologies if this is a dumb question.)

**Comment by Tratcher** [comment, marked as answer]
I think it's the same as any other .NET library. You either need to audit your project dependencies, or scan for the affected Yarp.ReverseProxy.dll on disk.

**Comment by Caradac** [comment]
I thought that might be the case.  Thanks.
